src/activity.c

```diff
@@ -894,6 +894,8 @@ static int cli_io_handler_show_tasks(struct appctx *appctx)
 	if (unlikely(sc_opposite(sc)->flags & SC_FL_SHUT_DONE))
 		return 1;
 
+	name_buffer = get_trash_chunk();
```

Comment:
Why split just this statement, but not `sc` and `tmp_activity`? Maybe it could be perceived as noise.